tests/test_cli.py

```diff
@@ -22,7 +23,22 @@
 from mlflow.exceptions import MlflowException
 from mlflow.entities import ViewType
 
-from tests.helper_functions import pyfunc_serve_and_score_model
+from tests.helper_functions import pyfunc_serve_and_score_model, get_safe_port
+from tests.tracking.integration_test_utils import _await_server_up_or_die
+
+
+@pytest.mark.parametrize("command", ["server", "ui"])
+def test_mlflow_server_command(command):
+    port = get_safe_port()
+    cmd = ["mlflow", command, "--port", str(port)]
+    process = subprocess.Popen(cmd)
+    try:
```

Comment:
do we want to validate that the subprocess executed successfully?
`assert process.check_returncode() == 0`
in the interests of a super-quick fail-fast indicator that there's something wrong with the build. This might be total overkill, though. Just a thought.

Reply:
`mlflow server` or `mlflow ui` usually never completes until we keyboard-interrupt 😅

Reply:
100% overkill ;) Thanks for the clarification! Let's merge it (you'll notice I followed this exact pattern in my PR)